ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadEIP6110Test.java

```diff
@@ -183,6 +185,8 @@ protected BlockHeader createBlockHeader(
             .excessDataGas(DataGas.ZERO)
             .dataGasUsed(100L)
             .depositsRoot(maybeDeposits.map(BodyValidation::depositsRoot).orElse(null))
+            .parentBeaconBlockRoot(
+                maybeParentBeaconBlockRoot.isPresent() ? maybeParentBeaconBlockRoot : null)
```

Comment:
```suggestion
                maybeParentBeaconBlockRoot.orElse(null)
```

Reply:
This won't work, as the parameter needs to be an Optional<Bytes32>

Reply:
that's a bit nasty